docs/getting-started/configure-integration-policy.asciidoc

```diff
@@ -0,0 +1,132 @@
+
+Memory threat protection levels are:
+
+* **Detect**: Detects memory threat activity on the host and generates an alert. The {agent} will **not** block the in-memory activity. Select the **Notify User** option to enable user notifications.
+* **Prevent** (Default): Detects memory threat activity on the host, forces the process or thread to stop, and generates an alert. User notifications are enabled by default. Deselect the **Notify User** option to disable them.
++
+TIP: Platinum and Enterprise customers can customize these notifications using the `{es-sec} {action} {rule}` syntax.
+
+[role="screenshot"]
+image::images/install-endpoint/memory-protection.png[Detail of memory protection section.]
+
+[discrete]
+[[behavior-protection]]
+== Malicious behavior protection
+
+Malicious behavior protection detects and stops threats by monitoring the behavior of system processes for suspicious activity. Behavioral signals are much more difficult for adversaries to evade than traditional file-based detection techniques.
+
+Malicious behavior protection is a paid feature and is enabled by default if you have a https://www.elastic.co/pricing[Platinum or Enterprise license]. If you upgrade to a Platinum or Enterprise license from Basic or Gold, malicious behavior protection will be disabled by default.
+
+Malicious behavior protection levels are:
+
+* **Detect**: Detects malicious behavior on the host and generates an alert. The {agent} will **not** block the malicious behavior. Select the **Notify User** option to enable user notifications.
+* **Prevent** (Default): Detects malicious behavior on the host, forces the process to stop, and generates an alert. User notifications are enabled by default. Deselect the **Notify User** option to disable them.
++
+TIP: Platinum and Enterprise customers can customize these notifications using the `{es-sec} {action} {rule}` syntax.
+
+[role="screenshot"]
+image::images/install-endpoint/behavior-protection.png[Detail of behavior protection section.]
+
+[discrete]
+[[event-collection]]
+== Event collection
+
+In the **Settings** section, review the events that {agent} will collect on each operating system. By default, all event data is collected. If you no longer want to collect specific type of event, deselect it.
+
+[role="screenshot"]
+image::images/install-endpoint/event-collection.png[Detail of event collection section.]
+
+[discrete]
+[[register-as-antivirus]]
+== Register {es-sec} as antivirus (optional)
+
+If you download the {agent} version 7.10 or later on Windows 7 or above, you can configure {es-sec} as your antivirus software by switching the **Register as antivirus** toggle on.
+
+[role="screenshot"]
+image::images/register-as-antivirus.png[Detail of Register as antivirus option.]
```

Comment:
How about we use one of these instead that hides the restrictions tooltip? First one has toggle on; second has toggle off. 
![2021-11-30_22-07-41](https://user-images.githubusercontent.com/57149392/144164885-d45cebda-d914-41bb-89d2-c9ec139207f6.png)

![2021-11-30_22-09-38](https://user-images.githubusercontent.com/57149392/144165066-f6508d72-6484-4e84-a3d2-576212f8da2a.png)

Reply:
I went with the second, since toggle off is the default state when you first create a policy.